scripts/release.py

```diff
@@ -415,8 +445,32 @@ def build_parser():
         default='ssh://git@github.com/rackerlabs/docs-rpc.git',
     )
     parser.add_argument(
+        '--branch', help='Branch to update'
+    )
+    parser.add_argument(
         '--tag', type=Tag, required=True, help='Name of new tag.'
```

Comment:
If you're hardcoding the tag into RELEASE_FILES you should either read tag from those files or validate that the two are equal.